scoring/templates/scoring/section.html

```diff
@@ -133,7 +133,7 @@ <h3 class="fs-4 mb-0 text-center">
     <div class="mb-2 d-flex align-items-center justify-content-between">
         <h3 class="mb-0">Key takeaways</h3>
       {% if social_graphics.zip %}
-        <a class="btn btn-sm btn-outline-primary" href="{% static social_graphics.zip %}">Download all (.zip)</a>
+        <a class="btn btn-sm btn-outline-primary js-social-graphic-download" fileType="zip" href="{% static social_graphics.zip %}">Download all (.zip)</a>
```

Comment:
Another small thing, but it’d be neater (and keep the HTML5 valid) to use [data attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/data-*) for those file type and image ratio properties on the links – so `data-file-type` and `data-img-ratio`. You can then pull them out again in the JavaScript with `el.dataset.fileType` and `el.dataset.imgRatio`.